### PR TITLE
feat(docker): build app with Grunt when building docker image, closes #4

### LINKED ISSUE #4: grunt build automatically when building Docker image

### PR DESCRIPTION
**Pull request by peterthomassen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,32 +9,17 @@ Contribute
...
-To setup your dev environment,
-
-- install nodejs and node package manager, `sudo apt-get install git npm nodejs nodejs-legacy` 
-- install ruby compass CSS framework, `sudo apt-get install ruby-compass` 
-- install grunt, karma, bower globally, `sudo npm install -g grunt grunt-cli karma bower` 
-- clone and cd to git repository
-- `npm install`
-- `bower install`
-
-To locally serve the files, run `grunt serve`. To build the project
```
Comment (nils-wisiol) [inline README.md]: keep the first sentence

File: .dockerignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# ignore everything expect for dist 
-# (this increases build-performance)
-*
-!dist
+bower_components
+dist
+node_modules
+tools
```
Comment (nils-wisiol) [inline .dockerignore]: make sure build performance is alright

File: README.md [hunk 3]
```diff
@@ -9,32 +9,17 @@ Contribute
 In order to contribute to this repository, extend code and tests 
 as appropriate and use clean commits.
 
-To setup your dev environment,
```
Comment (nils-wisiol) [inline README.md]: this environment is still needed for development; this should be reflected in the README

File: README.md [hunk 4]
```diff
@@ -9,32 +9,17 @@ Contribute
...
 
-Before building the docker image, make sure your build is clean by
-deleting all temp data and building a new release:
-
-    rm -r .tmp dist
-    grunt build
+First, make sure to locate the ultima design template in the 
+`ultima/` directory.
 
 To build the docker image serving the static files using nginx, use
```
Comment (nils-wisiol) [inline README.md]: Is this really relevant? What's the use case for this procedure?

**Comment by peterthomassen** [comment]
updated `.dockerignore` and reworked `README.md`

**Pull request merged**